apps/web/src/app/[locale]/(dashboard)/[wsId]/inventory/promotions/form.tsx

```
@@ -0,0 +1,198 @@
+                  </Select>
+                </FormControl>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+        </div>
+
+        <Button type="submit" className="w-full" disabled={loading}>
+          {loading
+            ? t('common.processing')
+            : !!data?.id
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)